packages/server/test/index.test.tsx

```diff
@@ -278,3 +278,48 @@ describe('integration tests', () => {
     });
   });
 });
+
+describe('call()', () => {
+  type Context = {};
+  const router = trpc
+    .router<Context>()
+    .query('q', {
+      input: z.number(),
+      async resolve({ input }) {
+        return { input };
+      },
+    })
+    .mutation('m', {
+      input: z.number(),
+      async resolve({ input }) {
+        return { input };
+      },
+    })
+    .subscription('sub', {
+      input: z.number(),
+      async resolve({ input }) {
+        return new trpc.Subscription<{ input: typeof input }>({
+          start(emit) {
+            emit.data({ input });
+            return () => {
+              // noop
+            };
+          },
+        });
+      },
+    });
+
+  test('query()', async () => {
+    const data = await router.createCaller({}).query('q', 1);
+    expectTypeOf(data).toMatchTypeOf<{ input: number }>();
+  });
+  test('mutation()', async () => {
+    const data = await router.createCaller({}).mutation('m', 1);
+    expectTypeOf(data).toMatchTypeOf<{ input: number }>();
+  });
+  test('subscription()', async () => {
```

Comment:
I dont think there's a use-case for subscription when on server-side (which is the whole point of this)